apps/hash-frontend/package.json

```diff
@@ -73,6 +73,7 @@
     "next-seo": "6.1.0",
     "nextjs-progressbar": "0.0.16",
     "notistack": "2.0.5",
+    "pluralize": "8.0.0",
```

Comment:
Can you import `pluralize` from `isomorphic-utils` instead? We add some rules to fix some incorrect pluralizations it generates:
```ts
import { pluralize } from "@local/hash-isomorphic-utils/pluralize";
```